scylla-cql/src/frame/value.rs

```diff
@@ -805,78 +871,113 @@ impl<'b> ValueList for Cow<'b, SerializedValues> {
 // BatchValues impls
 //
 
-// Implement BatchValues for slices of ValueList types
-impl<T: ValueList> BatchValues for &[T] {
-    fn len(&self) -> usize {
-        <[T]>::len(*self)
+impl<'r, IT> BatchValuesGatWorkaround<'r> for BatchValuesFromIterator<IT>
```

Comment:
To fix before merge: this should use a separate type (that is just a newtype wrapper)